drltrace/CMakeLists.txt

```diff
@@ -133,29 +137,45 @@ if (BUILD_TOOL_TESTS)
   get_target_path_for_execution(drltrace_path drltrace)
   get_target_path_for_execution(app_path drsyscall_app)
   prefix_cmd_if_necessary(drltrace_path OFF ${drltrace_path})
-
   add_test(drltrace ${drltrace_path} -- ${app_path})
   add_test(drltrace_libcalls ${drltrace_path} -logdir - -print_ret_addr -- ${app_path})
   add_test(drltrace_symargs ${drltrace_path} -logdir - -num_max_args 4 -- ${app_path})
 
+  set(path_to_config ${INSTALL_BIN}/drltrace.config)
+
+  # copy drltrace.config required to test libcall arguments printing
```

Comment:
I don't understand this suggestion, currently I'm launching drltrace for tests from bin where we also copy configuration file. Do we want to copy config in build dir and launch drltrace from there ?